contracts/test/vault/oeth_vault.fork-test.js

```diff
@@ -116,7 +120,7 @@ forkOnlyDescribe("ForkTest: Vault", function () {
       log(`from asset balance before ${formatUnits(fromBalanceBefore, 18)}`);
       const toBalanceBefore = await toAsset.balanceOf(oethVault.address);
 
-      const tx = await oethVault
+      const tx = oethVault
```

Comment:
why drop the await?

Reply:
It isn't needed here. Only on the `expect()`

Reply:
I'd argue the other way around. Its not needed on the expect. :-)
I guess its personal preference.